### DISCUSSION TITLE
求一份Telegram内分离视频的分流规则

### DISCUSSION THREAD

**Discussion by hide3110** [blackmatrix7/ios_rule_script#1076] [Q&A]
在Telegram中观看视频会占用大量流量，不知道能否可以将其视频的规则单独提取出来。试了在使用Telegram的时候，只是观看视频或是对话聊天，均含有149.154.160.0/20,no-resolve和91.108.0.0/16,no-resolve两条规则，是否无法单独分离视频的规则？还望回复，谢谢！

**Comment by WongAlice** [comment]
你可以在Telegram 设置里面直接禁用视频和图片自动缓存啊。遇到问题不要搞错工具，用脑子锤钉子可不行。
